recipes/baked_beans.json

```
@@ -0,0 +1,18 @@
+{
+    "recipe_name": "Baked Beans",
+    "serves": "1",
+    "prep_time": "2 mins",
+    "cooking_time": "2 mins",
+    "ingredients": [
+        {
+            "unit_size": "1",
+            "unit_type": "tin",
+            "ingredient_name": "heinz baked beans"
```

```suggestion
            "ingredient_name": "Branston baked beans"
```
Branston beans are far superior to Heinz beans. The thick sauce and much more developed flavour really sets it apart. 

Source: https://www.dailymail.co.uk/news/article-4548674/Heinz-left-trailing-place-blind-taste-test.html